engine/src/main/java/com/datatorrent/stram/StramClient.java

```diff
@@ -448,6 +450,14 @@ public void startApplication() throws YarnException, IOException
       }
       String libJarsCsv = copyFromLocal(fs, appPath, localJarFiles.toArray(new String[]{}));
 
+      if (UserGroupInformation.isSecurityEnabled() && customKeystoreConfig != null && customKeystorePath != null) {
+        String[] customSSLConfArr = {customKeystoreConfig, customKeystorePath};
+        String customSSLFileConf = copyFromLocal(fs, appPath, customSSLConfArr);
+        String configFileName = (new java.io.File(customKeystoreConfig)).getName();
+        dag.getAttributes().put(Context.DAGContext.STRAM_HTTP_CUSTOM_CONFIG, configFileName);
```

Comment:
@tweise  "COSTUM" instead of "CUSTOM" would be a spelling error, not this. You could call it a naming error but not a spelling error. Here "_CUSTOM_" is used in the sense of opposite of default. If one doesn't use this property, the Stram web service will end up using the "default" Hadoop SSL keystore hence the "_CUSTOM_" in the name.

Having said all this, this is just a name we are debating - I would leave it to @devtagare and  reviewers/committers to decide and I will be okay with that.